src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
+import core.basesyntax.lotteryball.Lottery;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[3];
```

Comment:
why do we need array of Lotteries? Could we use one instance of Lottery class to generate 3 random bals?